test/test_transforms_v2_refactored.py

```diff
@@ -536,7 +536,8 @@ def test_functional(self, size, make_input):
             (F._resize_image_pil, PIL.Image.Image),
             (F.resize_image, tv_tensors.Image),
             (F.resize_bounding_boxes, tv_tensors.BoundingBoxes),
-            (F.resize_mask, tv_tensors.Mask),
+            (F.resize_mask, tv_tensors.SegmentationMask),
+            (F.resize_mask, tv_tensors.DetectionMasks),
```

Comment:
Do we need the extra parametrization? Will anything break if we ever have different kernels for SegmentationMask and DetectionMasks